### DISCUSSION TITLE
Bumping Lando Causes Error

### DISCUSSION THREAD

**Discussion by johnpicozzi** [Homebrew/discussions#2233] [Casks] [answered]
When running `brew bump-cask-pr --version 3.4.0 lando` I get the following error. 

`Failure while executing; git checkout --no-track -b bump-lando-3.4.0 origin/master exited with 128.`

What am I doing wrong?

**Comment by johnpicozzi** [comment]
Looks like Lando has been updated to the latest and great. However, I would still like to know why I got this error and how it could be avoided in the future. It seems like it's isolated to me seeing as [this PR](https://github.com/Homebrew/homebrew-cask/pull/111960) appears to have been created from the `dump-cask-pr` script.

**Comment by neutric** [comment, marked as answer]
With the amount of context provided it is impossible to get down to why this happened. Please open an [issue](https://github.com/Homebrew/homebrew-cask/issues/new?assignees=&labels=&template=01_bug_report.yml) next time to report this as a potential bug. The bug report template outlines steps to be taken which might already fix the problem, and if not, make you provide enough context to start investigating. Please mark this as answered.
